collections/_priorities/xr-01-make-action-map-system-available-to-webxr.md

```diff
@@ -0,0 +1,18 @@
+---
+last_updated: "2024-11-30"
+category: xr
+rank: 1
+created: "2024-11-30"
+stage:
+  # Must be dates (YYYY-MM-DD), empty if not there yet.
+  planning: ""
+  ongoing: ""
+  done: ""
+anchor: "make-action-map-system-available-to-webxr"
+title: "Make action map system available to WebXR"
+description: |
```

Comment:
For description, how about:

```
OpenXR uses the "OpenXR action map system" for handling input, whereas WebXR (and any other XR SDKs added via GDExtension) will use their own way of handling input. This complicates making a game that will work on both OpenXR and WebXR. We'd like to add a generalized XR action map system, which will use the OpenXR action map system on OpenXR, but provide a fallback implementation that can be used with any XR SDK.
```

Here's is the proposal for this: https://github.com/godotengine/godot-proposals/issues/6548